src/components/pages/Calendar.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import FullCalendar from '@fullcalendar/react';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+
+import * as constants from '../../constants';
+
+export default function Calendar(props) {
+  const { isAdmin } = props;
+
+  const [addEventText, setAddEventText] = useState('Add Event');
+  const [headerText, setHeaderText] = useState('today,prev,next,addEventButton');
+
+  const calendarRef = React.createRef();
+  // For Reactiveness
+  const updateMedia = () => {
+    if (window.innerWidth > 768) {
+      calendarRef.current.getApi().changeView('timeGridWeek');
+      setAddEventText('Add Event');
+      setHeaderText('today prev,next addEventButton');
+    } else {
+      calendarRef.current.getApi().changeView('timeGridDay');
+      setAddEventText('+');
+      setHeaderText('today,prev,next,addEventButton');
```

Comment:
The header is a bit cramped on mobile right now, I think it might look a bit better if we remove the today button and put the addEvent button somewhere else.
```suggestion
      setHeaderText(prev,next');
```

Reply:
I thought about removing the today button, but didn't take action because it seems semi-important. I think I'll go ahead and remove that, but keep the addEventButton